src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -91,7 +92,16 @@ class FrameworkBundle extends Bundle
 {
     public function boot()
     {
-        ErrorHandler::register(null, false)->throwAt($this->container->getParameter('debug.error_handler.throw_at'), true);
+        if (class_exists(SymfonyRuntime::class)) {
```

Comment:
When symfony/runtime is installed, we assume it will register the ErrorHandler when appropriate.
That's not the case right now, but this PR fixes it.